### PR TITLE
[ELY-2070] CredentialStore is not able to get data from credential storage in file on IBM JDK 8

### PR DESCRIPTION
**Pull request by Ashpan**
https://issues.redhat.com/browse/ELY-2070

### CODE REVIEW

**Comment by Skyllarr** [comment]
@Ashpan Just a minor, I read Ashley's comment linked in this jira that these errors seem to be because IBM JDK is unable to read the ECDSA keys from a credential store. Is there some bug report or maybe release notes from IBM now that mention this which could be linked?

**Comment by Ashpan** [comment]
@Skyllarr I wasn't able to find any specific documentation that IBM has released stating that reading ECSDA keys doesn't work from a credential store, and their bug reporting systems seems to be something for clients/customers not for developers.

Although [I found this report](https://www.ibm.com/support/pages/apar/IJ15047), but it seems to be for "IBM JCE & JCEPlus providers" which doesn't seem like the issue we are facing.

**Comment by Skyllarr** [comment]
> @Skyllarr I wasn't able to find any specific documentation that IBM has released stating that reading ECSDA keys doesn't work from a credential store, and their bug reporting systems seems to be something for clients/customers not for developers.
> 
> Although [I found this report](https://www.ibm.com/support/pages/apar/IJ15047), but it seems to be for "IBM JCE & JCEPlus providers" which doesn't seem like the issue we are facing.

@Ashpan  The linked issue is probably not this problem since the link says it was fixed in 8.0.5.35, but this bug was originally found for 8.0.6.10 ( https://github.com/wildfly-security/wildfly-elytron/pull/1347#discussion_r420842788 ). 

Might be good to double check if the exception happen in isolation as well.

**Comment by Skyllarr** [comment]
@Ashpan But actually given this is JDK 8 which is basically deprecated  I am approving

**Review by Skyllarr — APPROVED** [review]

File: tool/src/test/java/org/wildfly/security/tool/CredentialStoreCommandTest.java [hunk 1]
```diff
@@ -316,6 +316,7 @@ public void testGenerateKeyPairDSA() throws CredentialStoreException {
 
     @Test
     public void testGenerateKeyPairECDSA() throws CredentialStoreException {
+        Assume.assumeFalse(IS_IBM); //Don't run if using IBM JDK https://issues.redhat.com/browse/ELY-2070
```
Comment (fjuma) [inline tool/src/test/java/org/wildfly/security/tool/CredentialStoreCommandTest.java:319]: @Ashpan Would you be able to create a new ELY issue for unignoring these tests on IBM JDK and then reference that ELY issue in the comment here.

Comment (Ashpan) [reply tool/src/test/java/org/wildfly/security/tool/CredentialStoreCommandTest.java]: Done! https://issues.redhat.com/browse/ELY-2214

**Review by fjuma — APPROVED** [review]

**Pull request merged**